test/uploads/e2e.spec.ts

```diff
@@ -76,7 +76,7 @@ describe('uploads', () => {
     await wait(110)
     const field = page.locator('.cell-image')
 
-    await expect(field).toContainText('image.png')
+    await expect(field).toContainText('<No Image>')
```

Comment:
Why was this one changed? Seems like `<No Image>` would contradict the name of the test.